sdk/storage/azdatalake/file/responses.go

```diff
@@ -245,11 +245,13 @@ func FormatDownloadStreamResponse(r *blob.DownloadStreamResponse, rawResponse *h
 		newResp.Version = r.Version
 		newResp.VersionID = r.VersionID
 	}
-	if val := rawResponse.Header.Get("x-ms-encryption-context"); val != "" {
-		newResp.EncryptionContext = &val
-	}
-	if val := rawResponse.Header.Get("x-ms-acl"); val != "" {
-		newResp.AccessControlList = &val
+	if rawResponse != nil {
```

Comment:
This change is _not_ required. Please see https://github.com/Azure/azure-sdk-for-go/issues/25345#issuecomment-3348050020 for more info.